src/material-experimental/mdc-button/fab.ts

```diff
@@ -49,27 +63,76 @@ export class MatFabButton extends MatButtonBase {
   // The FAB by default has its color set to accent.
   color = 'accent' as ThemePalette;
 
+  private _extended: boolean;
+  get extended() { return this._extended; }
+  set extended(extended: any) { this._extended = coerceBooleanProperty(extended); }
```

Comment:
The type here should still be boolean, but the class should define an `ngAcceptInputType` to `BooleanInput` (search for other `ngAcceptInputType` instances for examples)